code/datums/dna.dm

```diff
@@ -1,3 +1,41 @@
+///Setter macro used to modify dna blocks. For convenience, use SET_UI_BLOCK for identity and SET_UF_BLOCK for features.
+#define SET_DNA_BLOCK(string, blocknum, input, len_by_block) string = copytext(string, 1, len_by_block[blocknum]) + input + (blocknum < len_by_block.len ? copytext(string, len_by_block[blocknum+1]) : 0)
+
+#define SET_UI_BLOCK(string, blocknum, input) SET_DNA_BLOCK(string, blocknum, input, GLOB.total_ui_len_by_block)
+#define SET_UF_BLOCK(string, blocknum, input) SET_DNA_BLOCK(string, blocknum, input, GLOB.total_uf_len_by_block)
+
+/**
+ * List of unique indentity blocks whose length is different than the default dna block size.
+ * IF YOU ADD ANY UNIQUE IDENTITY BLOCK WHOSE SIZE IS DIFFERENT THAN DNA_BLOCK_SIZE, INCLUDE IT HERE.
```

Comment:
This comment is somewhat steeped in jargon, how likely are people to try and do this but not know what you mean